src/libs/Pusher/pusher.js

```diff
@@ -358,6 +359,20 @@ function reconnect() {
     socket.connect();
 }
 
+/**
+ * Combines channel parts and adds Pusher suffix when necessary
+ *
+ * @param {*} parts
+ *
+ * @returns {String}
+ */
+function getChannelName(...parts) {
+    return _.chain([...parts, CONFIG.PUSHER.SUFFIX])
+        .filter(Boolean)
```

Comment:
Is this doing something different than `_.compact()` would? I'd prefer `_.compact()` if it's just to remove falsy values.